frontend/src/pages/device/index.vue

```diff
@@ -38,7 +38,7 @@
                                 <ExternalLinkIcon />
                             </span>
                         </a>
-                        <button v-else data-action="open-editor" class="ff-btn ff-btn--secondary" disabled>
+                        <button v-else v-ff-tooltip:left="device.mode === 'developer' ? 'Please enable \'Editor Access\' in the \'Developer Mode\' tab' : '\'Developer Mode\' must be enabled'" data-action="open-editor" class="ff-btn ff-btn--secondary" disabled>
```

Comment:
Ill merge as is, but it seems like the model should have a method `InDeveloperMode()`

Reply:
shouldn't have merged - you're correct